rsa/prime.py

```diff
@@ -103,13 +90,32 @@ def randomized_primality_testing(n, k):
 def is_prime(number):
     """Returns True if the number is prime, and False otherwise.
 
+    >>> is_prime(2)
+    True
     >>> is_prime(42)
     False
     >>> is_prime(41)
     True
+    >>> [x for x in range(901, 1000) if is_prime(x)]
```

Comment:
This should be moved to a unit test, as it doesn't show any extra information for the user.